rust/common-ifc/src/error.rs

```diff
@@ -0,0 +1,19 @@
+use thiserror::Error;
+
+/// Result type for [PolicyError].
+pub type Result<T> = ::core::result::Result<T, PolicyError>;
+
+/// Errors for policy validation.
+#[derive(PartialEq, Error, Debug)]
+pub enum PolicyError {
```

Comment:
I kind of like the pattern of scoping the top-level error to the crate e.g., `CommonRuntimeError` for `common-runtime`. Presumably it is possible to have non-policy-specific errors in the crate, and it would be kind of annoying to have to juggle a second error enum to handle that case.